### ISSUE TITLE
Questions for the restriction in the source program

### ISSUE THREAD

**Issue by Sung-HwanLee** [snu-sf-class/swpp202201#36]
Please ask questions about the restriction in the source program by replying to this issue.

**Comment by strikef** [comment]
You're right; the PR restriction applies to both existing and newly implemented pass. That means you can implement up to 2 existing passes in a single sprint if you wish to, as long as you haven't implemented any of them in the previous sprints.

**Comment by taejung1205** [comment]
> You're right; the PR restriction applies to both existing and newly implemented pass. That means you can implement up to 2 existing passes in a single sprint if you wish to, as long as you haven't implemented any of them in the previous sprints.

If I add existing implementation, should this also meet the PR line diff condition (PR line diff should be around 200)? Implementing existing pass to the project simply requires addition of few lines.

**Comment by strikef** [comment]
@taejung1205

The statement in the project document seems misleading.
> The line diff of each PR should not exceed 200 except the 'extended PR'

Would be more closer to our intention. I'll fix the document.

**Comment by Jhuni0123** [comment]
Is the source program only the code generated by c-to-ll script?

**Comment by sunghwanl** [comment]
@Jhuni0123 
Yes, it is, but consider every IR program that is compiled by the skeleton backend as a possible input program.
